ibis/backends/sql/compilers/duckdb.py

```diff
@@ -349,8 +349,12 @@ def visit_UnwrapJSONBoolean(self, op, *, arg):
         )
 
     def visit_ArrayConcat(self, op, *, arg):
-        # TODO(cpcloud): map ArrayConcat to this in sqlglot instead of here
-        return reduce(self.f.list_concat, arg)
+        return reduce(
```

Comment:
@cpcloud FYI in [new duckb releases](https://github.com/duckdb/duckdb/discussions/14685) we should be able to do list_concat(a, b, c, d, ...), but we probably want to wait a bit before switching to that so people can still use older duckdb versions.